docs/quick-start.md

```diff
@@ -52,4 +57,43 @@ Run smart contract test with `yarn hardhat:test`
 
 - Edit your smart contract `YourContract.sol` in `packages/hardhat/contracts`
 - Edit your frontend in `packages/nextjs/pages`
-- Edit your deployment scripts in `packages/hardhat/deploy`
\ No newline at end of file
+- Edit your deployment scripts in `packages/hardhat/deploy`
+
+## *Beta* Setup using ```npx create-eth@latest```
+
+For a simplified setup, Scaffold-ETH 2 offers a beta npx method that guides you interactively through the setup.
+
+1. Bootstrap the project:
+```
+npx create-eth@latest
+```
+
+You will be presented with a series of prompts:
+
+- Your project name: Enter a name for your project, e.g., my-dapp-example.
+- What solidity framework do you want to use?: Choose your preferred solidity framework (Hardhat, Foundry)
+- Install packages?: Confirm if you want to install the necessary packages, typically by selecting Yes.
```

Comment:
![Screenshot 2023-08-28 at 1 12 31 PM](https://github.com/scaffold-eth/se2-docs/assets/80153681/8870776e-e6cc-49a4-9314-c56c57708936)

`nitpick:` I have seen begginers getting confused about how to select this, maybe we could say `hit enter` to select `yes` (defualt option) or type `n` and hit enter to select to `no` 

but lol yeah we could skip this if its too obvious

Reply:
great suggestion!! Will be changing this!